packages/xrpl/webpack.config.js

```diff
@@ -45,7 +46,12 @@ function getDefaultConfiguration() {
       }),
     ],
     module: {
-      rules: [],
+      rules: [
+        {
+          test: /\.js$/,
```

Comment:
Should we be checking for `.ts` files too? Or does this check the post-processed TS files that are converted to JS?

Reply:
The webpack config points at the dist folder as its entrypoint which contains only .js source code.  This rule allows the source maps of transpiled typescript to be reflected in the final sourcemaps.